pom.xml

```diff
@@ -14,6 +14,7 @@
 
         <gson.version>2.8.0</gson.version>
         <coremidi4j.version>0.9</coremidi4j.version>
+        <LX.version>29ac8d7364</LX.version>
```

Comment:
Not sure about this... doesn't this break the LX versioning scheme that should be consistent with the Maven release numbering for LX?

Or is this referencing a specific built version by jitpack? How will this generally get updated?

Reply:
This simply tells maven->jitpack to fetch the jars built from this commit: https://github.com/heronarts/LX/commit/29ac8d73640a7df4af8cecbb30d5ce2da8eebe43

Once we do a github release of LX with the Maven bits we can point this to 0.1.3 or whatever it is called. 

Specifying the LX version as `master-SNAPSHOT` has the same effect (for now since 29ac8d7364 is the latest master commit).

More info: https://jitpack.io/#heronarts/LX

Reply:
If one is doing local development of both LX and P3LX, then they would do a `mvn install` in the LX project which would publish an artifact to their local `~/.m2/repository` then here they would change the version to whatever got published locally. This is a bit clunky yes, but the solution would be to merge the 2 repos (artifacts could still be published separately)